From f7c0f3d8303f4c8b1140efaa0ecc122f49498e8a Mon Sep 17 00:00:00 2001
From: Hridya Valsaraju <hridya@google.com>
Date: Tue, 12 Jan 2021 12:45:50 -0800
Subject: [PATCH] Setup ueventd to support DMA-BUF system-secure heap

Memory allocated from this heap will be used for secure playback.

Test: manual
Bug: 175697666
Change-Id: I16275ef0cef39b7b56cdce7c3545622712d438dc
---
 rootdir/ueventd.rc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/rootdir/ueventd.rc b/rootdir/ueventd.rc
index a1e9b12ff..64a64d22b 100644
--- a/rootdir/ueventd.rc
+++ b/rootdir/ueventd.rc
@@ -48,6 +48,7 @@ subsystem dma_heap
 /dev/pmsg0                0222   root       log
 /dev/dma_heap/system      0444   system     system
 /dev/dma_heap/system-uncached      0444   system     system
+/dev/dma_heap/system-secure        0444   system     system
 
 # kms driver for drm based gpu
 /dev/dri/*                0666   root       graphics
-- 
2.25.1

